source/js/contact.js

```diff
@@ -15,9 +15,15 @@ $(document).ready(function(){
       subject = $("#subject").val().trim();
       text = $("#content").val().trim();
       $("#message").text("Sending E-mail");
-      $.get("https://" + site + "/api/contact", { email:email, to:to, name:name, subject:subject, text:text, recaptcha:recaptcha}, function(data) {
-        if(data === "sent") {
-          $("#message").empty().html("Email has been sent");
+      $.get(site + "/api/contact", { email:email, to:to, name:name, subject:subject, text:text, recaptcha:recaptcha}, function(data) {
+        if(!data.formSubmit) {
+          $("#message").empty().html('Error in recaptcha');
+        } else {
+          if (data.error) {
+            $("#message").empty().html(data.error);
+          } else {
+            $("#message").empty().html('Email Sent');
```

Comment:
You could simplify with a ternary 

`!data.formSubmit ? $("#message").empty().html(data.error) : $("#message").empty().html('Email Sent');`

That's a bit wordy so maybe something like 

```
var message = data.error ? data.error : 'Email Sent;
$("#message").empty().html(message);

```